### DISCUSSION TITLE
How can I specify the machine type and custom Dockerfile?

### DISCUSSION THREAD

**Discussion by HadiSDev** [getindata/kedro-vertexai#91] [Q&A] [answered]
Hi

I can't seem to find the documentation for specifying the machine type for the Kedro pipelines. 
Let say I have a node for training and I want to use 8xA100 for a specific node, how would that be done 

How do I specify the Dockerfile if I have a custom one?

**Comment by marrrcin** [comment, marked as answer]
Hi @HadiSDev !
1. You can use the configuration to specify resources required by your nodes like this (`run_config.resources` section) https://kedro-vertexai.readthedocs.io/en/0.7.0/source/02_installation/02_configuration.html
Right now, we don't support requesting of specific accelerators, but we're open to accept a pull request which will add this functionality (as per https://cloud.google.com/vertex-ai/docs/pipelines/machine-types - it will require adding a nodeSelector)
2. You can use custom `Dockerfile`, we're only suggesting `kedro-docker` for easy set up, you can modify the file later as you want.
